News: GRASS GIS 8.2.1 released (#346)
OSGeo/grass-website: content/news/2023_01_21_GRASS_GIS_8_2_1_released.md

---
title: "GRASS GIS 8.2.1 released"
date: 2023-01-21T11:25:51-04:00
layout: "news"
author: Markus Neteler
---

### What's new in a nutshell

The GRASS GIS 8.2.1 release offers over 80 improvements and bug fixes
compared to the previous release, 8.2.0. The update includes a range of
enhancements to the software's modules and graphical user interface.
Some notable module improvements include various bug fixes for `r.kappa`,
the addition of support for unregistering maps from other mapsets in
`t.unregister`, and a fix for cross-validation in `v.surf.rst`. The
graphical user interface has also seen improvements, with issues being
resolved for the database manager, the image group dialog, the import
of PostGIS and SQLite geometry data, the Postscript-based mapping tool,
and the Single-Window GUI toolbar. Additionally, a correction has been added

> to the Jupyter notebooks for bounding box reprojection for `InteractiveMap`

Reviewer: ```suggestion
to the grass.jupyter for bounding box reprojection within the `InteractiveMap` class
```

and the `GridModule` now cleans up temporary mapsets when an exception occurs.
Concerning the documentation, all addons now register their keywords in
the main keywords index, leading to a better overview of available
functionality. For mobile devices, viewport support and hamburger menu-style

> table of content has been added. The PROJ library support received

Reviewer: ```suggestion
table of content have been added. The PROJ library support received
```

important fixes for area bounding boxes and longitude estimation.
Compatibility fixes for PDAL 2.5.0 have been added, and updates have
been made to the available docker images, especially the base images
(Ubuntu and Debian). The Windows NSIS installer was also updated to fix

> rare installation problems. Overall, this release offers a more stable
> and efficient version of GRASS GIS for users.

Reviewer: ```suggestion
rare installation problems. Overall, this release offers a **more stable
and efficient version of GRASS GIS for users**.
```
